{
  try {
    myTestMethod.invoke(btt);
  }
 catch (  InvocationTargetException e) {
    throw e.getCause();
  }
}
